NotificationService/NotificationService.BusinessLibrary/Business/v1/EmailManager.cs

```diff
@@ -315,5 +315,12 @@ public async Task<IList<MeetingNotificationItemEntity>> CreateMeetingNotificatio
             this.logger.TraceInformation($"Completed {nameof(this.CreateNotificationEntities)} method of {nameof(EmailManager)}.", traceProps);
             return notificationEntities;
         }
+
+        /// <inheritdoc/>
+        public async Task<IList<EmailNotificationItemEntity>> GetEmailNotificationsByDateRange(string applicationName, DateTimeRange dateRange)
```

Comment:
Change these name, as you are getting only failed. If you want to make this method generic, accept filters as parameter and add status=failed as filter